src/utils/formatNumber.ts

```diff
@@ -0,0 +1,58 @@
+import config from 'config';
+
+const getUserLocale = (): string => {
+  return (
+    config?.locale ||
+    (typeof document !== 'undefined' && document.documentElement.lang) ||
+    (typeof navigator !== 'undefined' && navigator.language) ||
```

Comment:
Can follow the style of line 6 too, using ?.?. instead of typeof navigator !== 'undefined'

Reply:
On portal

document.documentElement.lang
>> 'en'
navigator.language
>> 'en-US'

Should we flip the check so that the specific comes 1st?